docs/tutorials/noise.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<p>latex is totally broken from <code>\rhos</code>  to the end. it seems missing <code>$</code> somewhere</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/quantum/pull/582/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>